sdk/maps/azure-maps-search/src/main/java/com/azure/maps/search/models/Testing.java

```diff
@@ -0,0 +1,88 @@
+package com.azure.maps.search.models;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.azure.core.http.HttpPipelineCallContext;
+import com.azure.core.http.HttpPipelineNextPolicy;
+import com.azure.core.http.HttpResponse;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.http.rest.Response;
+import com.azure.maps.search.MapsSearchClient;
+import com.azure.maps.search.MapsSearchClientBuilder;
+
+import reactor.core.publisher.Mono;
+
+public class Testing {
+    public static void main(String[] args) {
+        // build Client ID policy for use with Azure AD authentication
+        HttpPipelinePolicy clientIdPolicy = new HttpPipelinePolicy() {
+            @Override
+            public Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineNextPolicy next) {
+                return Mono.defer(() -> {
+                    context.getHttpRequest().setHeader("x-ms-client-id", System.getenv("CLIENT_ID"));
+                    return next.process();
+                });
+            }
+        };
+
+        // build subscription policy for use with Shared Key Authentication
+        HttpPipelinePolicy subscriptionKeyPolicy = new HttpPipelinePolicy() {
+            @Override
+            public Mono<HttpResponse> process(HttpPipelineCallContext context, HttpPipelineNextPolicy next) {
+                return Mono.defer(() -> {
+                    // context.getHttpRequest().setUrl(context.getHttpRequest().getUrl().toString() + "&subscription-key="
+                    //         + System.getenv("SUBSCRIPTION_KEY"));
+                    context.getHttpRequest().setUrl(context.getHttpRequest().getUrl().toString() + "&subscription-key="
+                            + "6f6kEhMW-khrzSkmqop49S6F3ZhP9wlmvliUJxi1AAo");
```

Comment:
This needs to be removed immediately as it's a security hole. This credential will have to be rotated since it's in GitHub's history now. Never add credentials to source code.